src/pngtocss.c

```diff
@@ -43,26 +40,32 @@ typedef enum {
 	bottom_right
 } point;
 
-typedef struct {
-	unsigned char *data;
-	png_uint_32 height;
-	png_uint_32 width;
-	int pxbytes;
-	png_bytep *row_pointers;
-} bmp;
+/* The super-class of a png_image, contains the decoded image plus additional
+ * data.
+ */
+typedef struct
+{
+	png_image image;
```

Comment:
can you add comments indicating that `height` and `width` are inside the `png_image` `struct`?